source/Node.Extensibility/Extensions/Infrastructure/Configuration/IExtensionConfigurationSettings.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Octopus.Node.Extensibility.Extensions.Infrastructure.Configuration
+{
+    public interface IExtensionConfigurationSettings : IHasConfigurationSettings
```

Comment:
What is the benefit of this interface?

Reply:
Good point, didn't end up needing that. Removing...